code/game/antagonist/antagonist_add.dm

```diff
@@ -2,14 +2,15 @@
 	if(!base_to_load)
 		return TRUE
 	. = FALSE
-	if(ispath(base_to_load))
+	if(base_to_load)
 		var/datum/map_template/base = base_to_load
 		base_to_load = null
-		if(SSmapping.map_templates[initial(base.name)])
-			base = SSmapping.map_templates[initial(base.name)]
-		if(!base)
-			base = new base()
-		else if(base.loaded > 0)
+		if(istext(base))
+			base = SSmapping.get_template(base)
+		if(!istype(base))
+			log_warning("Map template '[base]' could not be found, skipping load for [name].")
+			return FALSE
```

Comment:
I'd crash here so it fails unit tests. I know you have a unit test below which may also check this, though.

Reply:
Added a `PRINT_STACK_TRACE` instead of a `log_warning`. Unit test will fail already unless this is happening midround due to varediting.